root/utility/tableColumns.js

```diff
@@ -354,7 +355,7 @@ export function defineSeriesNumberColumn(
 }
 
 export function defineTextColumn<D>(
-  getText: (D) => string,
+  getText: (D) => StrOrNum,
```

Comment:
Related to the previous comments, it seems safer to just allow `string` to enforce formatting numbers for display.